### DISCUSSION TITLE
How to use component discovery add-on to create plain text translation components?

### DISCUSSION THREAD

**Discussion by AkiraVoid** [WeblateOrg/weblate#10137] [Q&A]
I have translations files stored by certain file structure: `translations/{language}/{pageNumber}.txt`, and every file looks like below:

```txt
stringA
stringB
stringC
```
I configured component discovery add-on with regular expression `translations/(?P<language>.+)/(?P<component>[0-9])\.txt`, and set monolingual base file to `translations/[originalLanguageTag]/{{ component }}.txt`, but get server error `KeyError: 'base_file_template'`. I want to know how I could corretly configure this add-on.

**Comment by nijel** [comment]
That sounds like a bug. Can you try upgrading? Posting server logs could be useful as well.
